lib/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="tech.relaycorp.relaydroid"
     >
 
+  <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Do we really need `android.permission.INTERNET` or is it a mistake?

Reply:
I believe we do need for PoWeb, but we can try taking it out and check if anything is blocked.

Reply:
Cool. Just created https://github.com/relaycorp/relaynet-endpoint-android/issues/43 to look into it.